core/state/database.go

```diff
@@ -29,25 +28,15 @@ import (
 	"github.com/ethereum/go-ethereum/ethdb"
 	"github.com/ethereum/go-ethereum/trie"
 	"github.com/ethereum/go-ethereum/trie/trienode"
-	exlru "github.com/hashicorp/golang-lru" //ex: external
+	//ex: external
```

Comment:
`//ex: external`
can delete now, it's added by me

Reply:
thx, updated